pennylane/devices/default_gaussian.py

```diff
@@ -641,14 +641,9 @@ class DefaultGaussian(Device):
             or iterable that contains unique labels for the subsystems as numbers (i.e., ``[-1, 0, 2]``)
             or strings (``['ancilla', 'q1', 'q2']``). Default 1 if not specified.
         shots (int): How many times the circuit should be evaluated (or sampled) to estimate
-            the expectation values.
-            If ``analytic == True``, then the number of shots is ignored
-            in the calculation of expectation values and variances, and only controls the number
-            of samples returned by ``sample``.
+            the expectation values. If `None`, the results are analytically computed and hence deterministic.
```

Comment:
```suggestion
            the expectation values. If ``None``, the results are analytically computed and hence deterministic.
```

Reply:
Between slack and github and rst and md I always end up so confused about the notation. :)